re.rizin.cutter.yml

```diff
@@ -0,0 +1,96 @@
+    add-ld-path: lib
+    merge-dirs: plugins;bin;lib
+    subdirectories: true
+    no-autodownload: true
+
+modules:
+  #stolen from https://github.com/flathub/org.freecadweb.FreeCAD/blob/master/org.freecadweb.FreeCAD.yaml
+  - name: pyside2
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_TESTS=OFF
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://download.qt.io/official_releases/QtForPython/pyside2/PySide2-5.15.5-src/pyside-setup-opensource-src-5.15.5.tar.xz
+        sha256: 3920a4fb353300260c9bc46ff70f1fb975c5e7efa22e9d51222588928ce19b33
+      - type: shell
+        commands:
+          - mkdir -p /app/include/qt5tmp && cp -R /usr/include/Qt* /app/include/qt5tmp # https://bugreports.qt.io/browse/PYSIDE-787
+          - sed -i 's|\(--include-paths=\)|\1/app/include/qt5tmp:|' sources/pyside2/cmake/Macros/PySideModules.cmake
+
+  - name: rizin
+    buildsystem: meson
+    config-opts:
+      - --buildtype=release
+      - -Drizin_plugins=extensions/lib/rizin/plugins
+      - -Dinstall_sigdb=true
+      - -Dsubprojects_check=false # we manually add sigdb subproject
+    sources:
+      - type: archive
+        url: https://github.com/rizinorg/rizin/releases/download/v0.4.1/rizin-src-v0.4.1.tar.xz 
+        sha256: 669d956b997820a36e423cf35d482aa99849254b9658cef6844459912dfdbfb8
+      - type: git
+        url: https://github.com/rizinorg/sigdb
+        branch: main
+        commit: a76430cbe2b4c41b5c5a041e42ba7532bd4a459b
+        dest: subprojects/sigdb
+
+  - name: graphviz
+    sources:
+      - type: archive
+        url: https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/6.0.2/graphviz-6.0.2.tar.gz
+        sha256: 4dccb9399080b7572570fa11b236c0305370bac569be7d121d66196409d76705
+
+  - name: cutter
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCUTTER_USE_BUNDLED_RIZIN=off
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCUTTER_ENABLE_KSYNTAXHIGHLIGHTING=on
+      - -DCUTTER_ENABLE_PYTHON=on
+      - -DCUTTER_ENABLE_PYTHON_BINDINGS=on
+      - -DSHIBOKEN_EXTRA_OPTIONS="--include-paths=/app/include/qt5tmp" # https://bugreports.qt.io/browse/PYSIDE-787
+      - -DCUTTER_EXTRA_PLUGIN_DIRS=/app/extensions/plugins
+    sources:
+      - type: archive
+        url: https://github.com/rizinorg/cutter/releases/download/v2.1.2/Cutter-v2.1.2-src.tar.gz
+        sha256: 088351e280d158e07498098ffba355a6272c2e4faf4b350ac7778e369eb23fe1
+      - type: patch
+        path: appstream.patch
+      - type: patch
+        path: desktop.patch
+    post-install:
+      - mkdir -p /app/extensions
+      - install -Dm644 --target=${FLATPAK_DEST}/share/metainfo/ ../src/re.rizin.cutter.appdata.xml
+      - cp ../src/img/cutter.svg ../src/img/re.rizin.cutter.svg
+      - install -Dm644 --target=${FLATPAK_DEST}/share/icons/hicolor/scalable/apps ../src/img/re.rizin.cutter.svg
```

Comment:
```suggestion
      - install -Dm644 ../src/img/cutter.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
```

Reply:
Fixed in https://github.com/flathub/flathub/pull/2251/commits/4249cea1c7be1c62c1847c2d3c773161079f9da1